gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineFixGradleJava.java

```diff
@@ -1,62 +0,0 @@
-/*
- * (c) Copyright 2020 Palantir Technologies Inc. All rights reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package com.palantir.baseline.plugins;
-
-import java.util.stream.Stream;
-import org.gradle.api.Plugin;
-import org.gradle.api.Project;
-import org.gradle.api.artifacts.ConfigurationContainer;
-import org.gradle.api.plugins.JavaBasePlugin;
-import org.gradle.api.plugins.JavaPluginConvention;
-import org.gradle.api.tasks.SourceSet;
-import org.gradle.api.tasks.SourceSetContainer;
-
-/**
- * Fixes up java configurations that are left resolvable and consumable for legacy reasons, in order to prepare for
- * Gradle 7 and to showcase code that resolves these when it shouldn't.
- *
- * <p>This will probably break some plugins in the wild, but we'd rather deal with that now than later.
- */
-public final class BaselineFixGradleJava implements Plugin<Project> {
```

Comment:
we should remove this in the [readme.md](https://github.com/palantir/gradle-baseline#compalantirbaseline-fix-gradle-java-off-by-default) as it is still mentioned and removal of this plugin caused excavator failure https://github.com/palantir/tritium/pull/1147

```
* Where:
Build file '/home/circleci/project/build.gradle' line: 36

* What went wrong:
A problem occurred evaluating root project 'tritium'.
> Plugin with id 'com.palantir.baseline-fix-gradle-java' not found.
```

https://github.com/palantir/gradle-baseline/blob/d2b734a0c6127f710930417d24b615c5cfb1d02f/README.md#L414-L426

Reply:
Good flag, will follow up with a PR!

Reply:
https://github.com/palantir/gradle-baseline/pull/1833